rmf_traffic_editor/gui/level.cpp

```diff
@@ -498,6 +498,48 @@ bool Level::delete_selected()
   return true;
 }
 
+bool Level::delete_used_entities()
```

Comment:
Would it be possible to change this function signature so it just receives the selected vertex as a parameter rather than having to iterate through all the vertices in the level to find the selected one?

Reply:
Yes, my mistake.

The selected vertex is already calculated before the call of this function, so it is redundant to do another loop to find the selected one.